app/src/main/res/values-fr/strings.xml

```diff
@@ -57,7 +57,7 @@
     <string name="nearest_accomodation"><u>Hôtel/Auberge:</u></string>
     <string name="need_account">L\'application ne peut fonctionner sans informations de connexion WarmShowers</string>
     <string name="no_results">Votre recherche n\'a donné aucun résultat.</string>
-    <string name="no_starred_hosts">Vous n\'avez présentement aucun hôte favori (Profils d\'hôtes disponibles hors ligne). Vous pouvez ajouter des hôtes favoris en cliquant l\'icône étoile sur leur page d\'information.</string>
+    <string name="no_starred_hosts">Vous n’avez présentement aucun hôte favori (Profils d’hôtes disponibles hors ligne). Vous pouvez ajouter des hôtes favoris en cliquant le bouton « Favori » sur leur page d’information.</string>
```

Comment:
Remove the spaces in "« Favori »" to be in line with the Spanish translation?

Reply:
Nope, Spanish uses the guillemets as quotation marks without a space, French, on the other hand, does use spaces.

Reply:
Wow, OK. Thank you.